var/spack/repos/builtin/packages/py-mkdocs/package.py

```diff
@@ -16,17 +16,30 @@ class PyMkdocs(PythonPackage):
 
     license("BSD-2-Clause")
 
+    version("1.5.3", sha256="eb7c99214dcb945313ba30426c2451b735992c73c2e10838f76d09e39ff4d0e2")
     version("1.3.1", sha256="a41a2ff25ce3bbacc953f9844ba07d106233cd76c88bac1f59cb1564ac0d87ed")
 
+    depends_on("python@3.7:", when="@1.4.1:", type=("build", "run"))
     depends_on("python@3.6:", type=("build", "run"))
+    depends_on("py-hatchling", when="@1.4.1:", type="build")
     depends_on("py-setuptools", type="build")
+    depends_on("py-click@7.0:", when="@1.4.1:", type=("build", "run"))
     depends_on("py-click@3.3:", type=("build", "run"))
+    depends_on("py-jinja2@2.11.1:", when="@1.4.1:", type=("build", "run"))
     depends_on("py-jinja2@2.10.2:", type=("build", "run"))
+    depends_on("py-markdown@3.2.1:", when="@1.5.0:", type=("build", "run"))
     depends_on("py-markdown@3.2.1:3.3", type=("build", "run"))
```

Comment:
```suggestion
    depends_on("py-markdown@3.2.1:3.3", type=("build", "run"), when="@:1.4")
```
Otherwise this constraint will also be applied to newer versions